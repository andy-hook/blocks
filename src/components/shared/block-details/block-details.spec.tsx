import React from "react"
import BlockDetails from "./block-details"
import { renderWithTheme } from "@test-utils"

jest.mock("react-truncate-string")

describe("<BlockDetails />", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme("dark", <BlockDetails />)
    expect(tree).toMatchSnapshot()
  })
})
