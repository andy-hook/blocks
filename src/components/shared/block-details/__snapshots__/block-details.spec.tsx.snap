// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockDetails /> renders correctly 1`] = `
<div
  className="sc-jTzLTM jeaEVH"
>
  <article
    className="sc-gZMcBi kxLsOT"
  >
    <header
      className="sc-htoDjs bzhYRm"
    >
      <div
        className="sc-iwsKbI ldERec"
      >
        <div
          className="sc-bdVaJa jfEAsh sc-dnqmqq dpESLn"
          size="lg"
        >
          <div
            className="sc-bwzfXH dXrMlk"
          >
             
          </div>
        </div>
        <div
          className="sc-htpNat bHaBzt"
          size="lg"
        >
          <div
            className="sc-bxivhb dsXjOH"
          >
             
          </div>
        </div>
      </div>
      <a
        className="sc-bZQynM cSTeFT"
        href="https://www.google.com"
        type="primary"
      >
        <span
          className="sc-gzVnrw ipIBGD"
        >
          View transactions
        </span>
      </a>
    </header>
    <ul
      className="sc-gqjmRU hnZXkc"
    >
      <div
        className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Size
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Difficulty
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Total Difficulty
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dTNUzo"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Used
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH dXrMlk"
            >
               
            </div>
          </div>
        </div>
      </div>
      <div
        className="sc-ifAKCX kEvjdA sc-VigVT bcVrsj"
      >
        <div
          className="sc-htpNat hwnZXv"
          size="sm"
        >
          Gas Limit
        </div>
        <div
          className="sc-EHOje foZbCf"
        >
          <div
            className="sc-bdVaJa htfwrO"
            size="lg"
          >
            <div
              className="sc-bwzfXH eAbjIM"
            >
               
            </div>
          </div>
        </div>
      </div>
    </ul>
  </article>
</div>
`;
